vowpalwabbit/core/src/reductions/csoaa_ldf.cc

```diff
@@ -643,6 +643,7 @@ base_learner* VW::reductions::csldf_setup(VW::setup_base_i& stack_builder)
       .add(make_option("csoaa_ldf", csoaa_ldf)
                .keep()
                .necessary()
+               .one_of({"m", "multiline", "mc", "multiline-classifier"})
```

Comment:
if ((ldf_arg == "singleline" || ldf_arg == "s") || (ldf_arg == "singleline-classifier" || ldf_arg == "sc"))

are these missing?

Reply:
if ((ldf_arg == "singleline" || ldf_arg == "s") || (ldf_arg == "singleline-classifier" || ldf_arg == "sc"))
      THROW(
          "ldf requires either m/multiline or mc/multiline-classifier.  s/sc/singleline/singleline-classifier is no "
          "longer supported");

Reply:
theyre spelled out just to throw an error